docs/schemas/resource/IfcRepresentationResource/Entities/IfcMapConversionScaled.md

```diff
@@ -1,16 +1,77 @@
 # IfcMapConversionScaled
 
-An _IfcMapConversionScaled_ is a type of _IfcMapConversion_ that along with an EPSG code supplies a constant scale factor for coordinate conversion. The usage is restricted to when constant scale factors are explicitly exchanged. Typically, these are relatively speaking smaller facilities such as buildings where a constant scale factor is agreed on a project.
+An _IfcMapConversionScaled_ is a type of _IfcMapConversion_ that supplies factors for coordinate conversion. The usage is restricted to when factors are explicitly exchanged. 
+
+> NOTE  Typically, these are relatively smaller facilities such as buildings where constant factors are agreed on a project.
+
+For this transformation, _IfcMapConversionScaled_ data are used for:
+1. a scaling of the three axes (x,y,z), by the same _IfcMapConversionScaled.Scale_
+2. a multiplication of the x-axis by _IfcMapConversionScaled.FactorX_
+3. a multiplication of the y-axis by _IfcMapConversionScaled.FactorY_
+4. a multiplication of the z-axis by _IfcMapConversionScaled.FactorZ_
+5. followed by an **anti-clockwise** rotation about the z-axis of $\theta$, where: 
+
+$$
+\theta=arctan\left(\frac{IfcMapConversionScaled.XAxisOrdinate}{IfcMapConversionScaled.XAxisAbscissa}\right)
+$$
+
+6. and then a translation in (x,y,z) of _IfcMapConversionScaled.Eastings_, _IfcMapConversionScaled.Northings_, _IfcMapConversionScaled.OrthogonalHeight_.
 
 > HISTORY New entity in IFC4X3_ADD1
 
+**Equations**
+
+Below are the relevant equations for _IfcMapConversionScaled_. The equations are given: a) in full expanded matrix form, useful for programmers to understand the exact sequence of operations, and to easily adapt to instead of multiply a standard CG (Computer Graphics) 4x4 matrix; b) in the simplified equation form, which is sufficient for calculating a single point.
+
+a) matrix form
```

Comment:
@aothms can you check the md rendering of the formulas? They render correctly in my md viewer, but apparently they don't here